_hackbars/Aryan_Bindal.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: hackbar
+title: Wash your hands and avoid touching your face.
+author: Aryan Bindal
+---
+
+Hello wonderful coders out there! This is my first participation in Hacktoberfest and I am excited to make this pull request.     
+It's time to give back to the community.
+
+---
+
+## Make Earth a better place
```

Comment:
Codacy found an issue: [[heading-style] Headings should use setext](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6303001)